### ISSUE TITLE
ReduceLROnPlateau has no get_last_lr

### ISSUE THREAD

**Issue by amueller** [pytorch/pytorch#110776]
### 🚀 The feature, motivation and pitch

``ReduceLROnPlateau`` has no method ``get_last_lr``, which seems to be the recommended method to get the current learning rate. I wonder if there's a reason why it's missing?

cc @vincentqb @jbschlosser @albanD @janeyx99 @crcrpar

**Comment by amueller** [comment]
Thanks for your reply. I'm on ``'2.1.0+cu121'``

**Comment by janeyx99** [comment]
Ah, I suppose this fix https://github.com/pytorch/pytorch/pull/108464 wasn't landed as a part of the release as it was too recent. Trying the same code on a nightly (install instructions on the toggle part of our pytorch.org page) should work.
